### DISCUSSION TITLE
Question: ETA of the 14 and 15 Feb meeting minutes?

### DISCUSSION THREAD

**Discussion by orthoxerox** [dotnet/csharplang#152] [General]
I'm especially interested in the meeting with Unity, since they are likely to ask for performance related stuff like readonly locals, ref operators, fixed size buffers and maybe even type classes.

**Comment by jcouv** [comment]
I see they were posted:
- https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-02-14.md
- https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-02-15.md

I'll go ahead and close the issue.

**Comment by orthoxerox** [comment]
They are still stubs with no content.

**Comment by jcouv** [comment]
Oops, I failed to notice that. Re-opened.

**Comment by gulshan** [comment]
Maybe this can be closed now.
